tests/helpers/has-attributes.js

```diff
@@ -0,0 +1,16 @@
+import QUnit from 'qunit';
+
+function compareArrays(attributes, attributesToTest){
+  attributesToTest.sort();
+
+  return attributes.sort().every(function(element, index) {
+    return element === attributesToTest[index]; 
+  });
+}
```

Comment:
This looks good, just check your formatting, like `){` => `) {`, and that trailing semicolon.

Would also consider adding YUIdoc documentation to this function.